src/targets/services/keepIndexFullNameAndDisplayNameUpToDate.js

```diff
@@ -18,7 +19,11 @@ export const keepIndexFullNameAndDisplayNameUpToDate = async () => {
   )
   const contactsToUpdate = await fetchContactsToUpdate(
     client,
-    normalizedService.current_state.last_success
+    get(
+      normalizedService,
+      'current_state.last_success',
+      '0000-01-01T00:00:00.00Z'
```

Comment:
Comme le service ne traite que les contacts plus récents qu'une certaine date, mettre la date à "0" permet de traiter tous les contacts (dans le cas ou le service n'a jamais été exécuté)

Reply:
Nit: From the point of the reader, it is a bit hard by reading the function call to understand that the 3rd attribute of fetchContactsToUpdate is the minimum update date.